spec.emu

```diff
@@ -266,8 +266,8 @@ contributors: Ujjwal Sharma, Younies Mahmoud
           1. Let _displaySlot_ be the Display Slot value.
           1. Let _valueSlot_ be the Value Slot value.
           1. Let _unit_ be the Unit value.
-          1. Let _style_ be the current value of the _styleSlot_ slot of _durationFormat_.
-          1. Let _nextStyle_ be the current value of the _nextStyleSlot_ slot of _durationFormat_.
+          1. Let _style_ be _durationFormat_.[[&lt;_styleSlot_&gt;]].
+          1. Let _nextStyle_ be _durationFormat_[[&lt;_nextStyleSlot_&gt;]].
```

Comment:
This is missing a dot before `[[` here and below. Later commits fix these errors.